content/INTRODUCTION.md

```diff
@@ -0,0 +1,18 @@
+**Requirements**: No programming experience, at least 12 years old
+
+**Duration**: 
+
+**Standards**: K-12 Computer Science Framework Core Concepts and Practices
+* Algorithms and Programming
+  - Collaborating Around Computing
+  - Recognizing and Defining Computational Problems
+  - Developing and Using Abstractions
+  - Creating Computational Artifacts
+  - Testing and Refining Computational Artifacts
+
+## Web Development Curriculum
+
+Learn to code and start building professional, career-boosting websites and web applications.
```

Comment:
Place a comma `,`  after the text `websites`
`Learn to code and start building professional, career-boosting websites, and web applications.`

Reply:
Sorry, but a comma `,` between `websites` and `and web applications` is not necessary since we only have two subjects here the `website` and `web applications` which is joined by `and`. Please check. Thanks.